ansible/roles/bie-hub/templates/bie-hub-config.properties

```diff
@@ -0,0 +1,33 @@
+bie.index.url={{ bie_index_url | default('http://localhost:8090/bie-index') }}
```

Comment:
To be honest, I'm not a fan of having default values for properties like this. I would rather see a build error or a runtime failure if something isn't set than to have the system point to an unexpected service. That's a broader issue though.